File: backend/src/main/java/com/lawencon/ticketms/dto/comment/CommentInsertReqDto.java
package com.lawencon.ticketms.dto.comment;

import java.util.List;

public class CommentInsertReqDto {
	private String comments;
	private Long ticketId;
	private Long userId;
	private List<FileInsertReqDto> file;
	public String getComments() {
		return comments;
	}
	public void setComments(String comments) {
		this.comments = comments;
	}
	public Long getTicketId() {
		return ticketId;
	}
	public void setTicketId(Long ticketId) {
		this.ticketId = ticketId;
	}
	public Long getUserId() {
		return userId;
	}
	public void setUserId(Long userId) {
		this.userId = userId;
	}
	public List<FileInsertReqDto> getFile() {
		return file;
	}
	public void setFile(List<FileInsertReqDto> file) {
		this.file = file;
	}
}

File: frontend/src/app/dto/comment/comment.insert.req.dto.ts
import { FileInsertReqDto } from "./file.insert.req.dto"

export class CommentInsertReqDto {

    private comments! : string 
	private ticketId! : number
	private userId! : number
    private file! : FileInsertReqDto[]
    
    public get _comments(): string {
        return this.comments;
    }

    public set _comments(_comments: string) {
        this.comments = _comments;
    }

    public get _ticketId(): number {
        return this.ticketId;
    }

    public set _ticketId(_ticketId: number) {
        this.ticketId = _ticketId;
    }

    public get _userId(): number {
        return this.userId;
    }

    public set _userId(_userId: number) {
        this.userId = _userId;
    }

    public get _file(): FileInsertReqDto[] {
        return this.file;
    }

    public set _file(_file: FileInsertReqDto[]) {
        this.file = _file;
    }

}